massa-models/src/block.rs

```diff
@@ -357,6 +358,117 @@ impl Deserializer<Block> for BlockDeserializer {
     }
 }
 
+/// Deserializer for `Block`
+pub struct BlockDeserializerLW {
```

Comment:
I don't understand why you have implemented this which is the same code as BlockDeserializer just use the lightweight deserializer for header. Why not using it directly on block deserializer ?